xmtp_mls/src/groups/mls_sync.rs

```diff
@@ -1708,6 +1712,7 @@ where
                 version: Some(GroupMessageInputVersion::V1(GroupMessageInputV1 {
                     data: payload.to_vec(),
                     sender_hmac: sender_hmac.into_bytes().to_vec(),
+                    should_push: Some(false),
```

Comment:
Not really sure how to get the should push in this case? Or what case this is...

Reply:
@codabrink do you know what case this is?

Reply:
I think we should put should_push in the group_intent table rather than the group_messages.